### DISCUSSION TITLE
[Release] Version 0.4.0b8

### DISCUSSION THREAD

**Discussion by Herklos** [Drakkar-Software/OctoBot#1645] [General]
Hello everyone,
The 0.4.0b8 version of OctoBot is released ! It includes a tutorial for new users, improvements to the web interface, automated updates for binary versions and many exchanges issues fixes.

Available at : https://github.com/Drakkar-Software/OctoBot/releases/tag/0.4.0b8
![PEzSHeL](https://user-images.githubusercontent.com/10454670/114266013-3bd7cb00-99f4-11eb-9878-6467f7873cc5.png)

**Comment by nathanp83** [comment]
Is there an easy way to upgrade from 0.4.0a24 to this?

**Reply by Herklos** [reply]
Hi @nathanp83, what's your installation method ? You downloaded 0.4.0a24 binary ?

**Reply by nathanp83** [reply]
Hello @Herklos, Sorry I failed to supply more information.  I had received 0.4.0a24 for Windows 10 because 0.4.0 had a bug with Binance.us.

For this upgrade(0.4.0b8) I created a new directory and copied over just config files from the old install.  A few tweaks and I was up and running again.  I love updates!

**Reply by Herklos** [reply]
Great ! :+1:

**Comment by aburchak** [comment]
Hi Herklos. What is the file type of OctoBot_macos_x64? It has no extension, so I've tried to add .dmg, pkg, app, zip... Nothing works

**Reply by Herklos** [reply]
Hi @aburchak, you should open a new terminal window and type `chmod +x OctoBot_macos_x64` then `./OctoBot_macos_x64`

**Reply by aburchak** [reply]
It works, thanks a lot!
